polyfill/test/validStrings.mjs

```diff
@@ -298,15 +298,19 @@ const timeZoneUTCOffsetName = seq(
   hour,
   choice([minuteSecond, [minuteSecond, [fraction]]], seq(':', minuteSecond, [':', minuteSecond, [fraction]]))
 );
-const timeZoneBracketedName = withCode(
-  choice(timeZoneUTCOffsetName, ...timezoneNames),
-  (data, result) => (data.ianaName = ES.GetCanonicalTimeZoneIdentifier(result).toString())
-);
-const timeZoneBracketedAnnotation = seq('[', timeZoneBracketedName, ']');
+const unpaddedHour = choice(
+  digit(),
+  seq(character('1'), digit()),
+  seq(character('2'), choice(character('0'), character('1'), character('2'), character('3')))
+)
 const timeZoneIANAName = withCode(
-  choice(...timezoneNames),
+  choice(
```

Comment:
I checked and actually the `Etc/GMT...` time zones are already present in the timezones.json package, so they'll be included here in `choice(...timezoneNames)`. So maybe we can just leave this out and it'll be sufficient to add a comment here explaining why the validStrings deviates from the grammar in the spec in this case.

Reply:
comment added